tests/countries/test_turkmenistan.py

```
@@ -0,0 +1,86 @@
+        self.assertIn(eid_al_fitr_2024, self.holidays_2024)
+        self.assertIn("Eid al-Fitr", self.holidays_2024[eid_al_fitr_2024])
+        
+        self.assertIn(eid_al_adha_2024, self.holidays_2024)
+        self.assertIn("Eid al-Adha", self.holidays_2024[eid_al_adha_2024])
+        
+        eid_days = [d for d, name in self.holidays_2024.items() if "Eid al-Fitr" in name or "Eid al-Adha" in name]
+        self.assertGreaterEqual(len(eid_days), 6)
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)